README.md

```diff
@@ -106,13 +86,14 @@ repo.archive.readFile('/example.txt', 'utf-8', (err, data) => {
 
 Creates a new dat object. The options passed here will be default for any dats created using the `add` method.
 
- * `options`: any options you can pass to [mafintosh/hyperdrive](https://github.com/mafintosh/hyperdrive). These options will become default for all dats. In addition it has the following:
-  * `signalhub`: An optional string or array of strings for [signalhubws](https://github.com/soyuka/signalhubws) servers to use for WebRTC. Note that this isn't compatible with older `signalhub` servers. These servers are used to discover and connect to WebRTC peers. Note that each signalhub constitutes a different WebRTC swarm. The default is `https://signalhubws.mauve.moe/`
-  * `gateway`: An optional string or array of strings for [dat-gateway](https://github.com/garbados/dat-gateway/) instances for websocket replication. This is used to proxy data from the rest of the dat network when there are no WebRTC peers available with your data. There is no gateway set by default, but you can use `https://gateway.mauve.moe/` if you want it.
+ * `options`: any options you can pass to [mafintosh/hyperdrive](https://github.com/mafintosh/hyperdrive). These options will become default for all dats. It also gets passed as options into [discovery-swarm-web](https://github.com/RangerMauve/discovery-swarm-web). In addition it has the following:
+  * `persist`: Whether the data should persist locally or load in memory. Default: `false` (memory only). This uses [random-access-web](https://github.com/RangerMauve/random-access-web) for persistence to choose the best storage layer for the current browser.
+  * `db`: Pass in the random-access-storage](https://github.com/random-access-storage/random-access-storage) instance to use. (overrides the `persist` option)
+  * `id`: The ID to use when replicating hyperdrives
 
 ### `dat.get(url, [options])`
 
-Adds a new dat with the given url. Joins the appropriate swarm for that url and begins to upload and download data. If the dat was already added, it will return the existing instance. One gotcha is that dat-js doesn't support DNS resolution yet. As such you'll need to use the actual archive key for loading websites.
+Adds a new dat with the given url. Joins the appropriate swarm for that url and begins to upload and download data. If the dat was already added, it will return the existing instance. One gotcha is that dat-js doesn't support DNS resolution yet. As such you'll need to use the actual archive key for loading websites. `dat-js` adds a `url` field to the archive, but you can see the rest of the APIs available in the [hyperdrive](https://www.npmjs.com/package/hyperdrive) docs.
 
  * `url`: Either a `dat://` url or just the public key in string form.
```

Comment:
Been trying to update `public key` -> `read key`. Not sure what to do with url though... (need to really get that nomeclature draft in)

Reply:
I added a thing to mention that the `url` is the `read key`